packages/server/src/wallets/handlers/get_wallet_state_and_history.rs

```diff
@@ -1,104 +1,53 @@
-use firefly_client::models::Deploy;
-use firefly_client::{BlocksClient, ReadNodeClient, template};
+use chrono::{DateTime, Utc};
+use firefly_client::ReadNodeClient;
 
-use crate::common::tracing::record_trace;
-use crate::wallets::models::{
-    Direction,
-    Operation,
-    Transfer,
-    WalletAddress,
-    WalletStateAndHistory,
-};
+use crate::wallets::contracts::{create_check_balance_contract, get_user_history_contract};
+use crate::wallets::dtos::{BlockChainTransactionRecord, Transaction};
+use crate::wallets::models::{Direction, Transfer, WalletAddress, WalletStateAndHistory};
 
-template! {
-    #[template(path = "wallets/check_balance.rho")]
-    #[derive(Debug, Clone)]
-    struct CheckBalance {
-        wallet_address: WalletAddress,
-    }
-}
-
-#[tracing::instrument(
-    level = "info",
-    skip_all,
-    fields(address),
-    err(Debug),
-    ret(Debug, level = "trace")
-)]
+#[tracing::instrument(level = "trace", skip_all, ret(Debug), err(Debug))]
```

Comment:
this did you change that to trace? we want be getting any spans on info level now. lets keep it as it was
```suggestion
#[tracing::instrument(
    level = "info",
    skip_all,
    fields(address),
    err(Debug),
    ret(Debug, level = "trace")
)]
```